scripts/lite-fhir-schema.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const fs = require("fs");
+const fhir = require("../src/sg/gov/moh/fhir/4.0.1/schema.json");
+
+const filename = "src/sg/gov/moh/fhir/4.0.1/lite-schema.json";
+
+const resources = ["Bundle", "Device", "Observation", "Organization", "Patient", "Practitioner", "Specimen"];
+const resourceQueue = [...resources];
+
+const liteSchema = {
+  $schema: "http://json-schema.org/draft-07/schema#",
+  $id: "https://schemata.openattestation.com/sg/gov/moh/fhir/4.0.1/lite-schema.json#",
+  // id: "http://hl7.org/fhir/json-schema/4.0",
+  title: "Lite FHIR schema for Notarise.gov.sg HealthCerts",
+  description: "see http://hl7.org/fhir/json.html#schema for information about the FHIR Json Schemas",
+  $ref: "#/definitions/Bundle",
+  definitions: {}
+};
+
+/** ====== Helper Functions ====== **/
+
+function stringToRef(ref = "") {
+  return ref.replace("#/definitions/", "");
+}
+
+function findUniqueNestedReferences(obj = {}, searchKey = "$ref", results = []) {
```

Comment:
searchKey looks useless ? never used anywhere and the function states we search for references